### DISCUSSION TITLE
Expose SRAM for GBA games

### DISCUSSION THREAD

**Discussion by Blazekickn** [RetroAchievements/RAIntegration#1020] [Completed]
Bryan and I recently looked into a game, [Choro Q Advance](https://retroachievements.org/game/6875). This game directly writes most important data (such as race flags and parts) into SRAM rather than storing them in WRAM like most games, as proven by looking in Bizhawks RAM Search. A set could probably still be made by making stuff one session or checking a mirror of some (but not all!) of the data that only shows up on a specific screen id, but not having the ability to check SRAM for achievements heavily cripples the ability to develop achievements for this game.

**Comment by Blazekickn** [comment]
Probably worth exposing other potentially useful data too, such as VRAM or the ROM Header

**Comment by Jamiras** [comment]
Implemented by https://github.com/RetroAchievements/rcheevos/pull/265, which was included in the 1.3.0 DLL.
